src/test/data/tx_invalid.json

```diff
@@ -185,5 +185,21 @@
 [[["0000000000000000000000000000000000000000000000000000000000000100", 0, "HASH160 0x14 0xc5b93064159b3b2d6ab506a41b1f50463771b988 EQUAL"]],
 "0100000001000100000000000000000000000000000000000000000000000000000000000000000000030251b1000000000100000000000000000000000000", "P2SH,CHECKLOCKTIMEVERIFY"],
 
+["Failure due to non-matching CHECKLOCKTIMEVERIFY kind"],
+[[["0000000000000000000000000000000000000000000000000000000000000100", 0, "1"]],
+"01000000010001000000000000000000000000000000000000000000000000000000000000000000000251b100000000010000000000000000000065cd1d", "P2SH,CHECKLOCKTIMEVERIFY"],
+
+["Failure due to an empty stack for CHECKLOCKTIMEVERIFY"],
+[[["0000000000000000000000000000000000000000000000000000000000000100", 0, "1"]],
+"010000000100010000000000000000000000000000000000000000000000000000000000000000000001b1010000000100000000000000000000000000", "P2SH,CHECKLOCKTIMEVERIFY"],
```

Comment:
Duplicate of https://github.com/eordano/bitcoin/blob/test/cltv/src/test/data/tx_invalid.json#L139, but triggered in the scriptSig instead of scriptPubKey. How about you move this test to be next to that one to keep the similar testcases together?